### ISSUE TITLE
The "x-ms-creation-time" header is missing in GetBlob response

### ISSUE THREAD

**Issue by kshcherbatov** [Azure/Azurite#963]
### Which service(blob, file, queue, table) does this issue concern?
blob
### Which version of the Azurite was used?
3.13.1 (seems to be also actual at trunk)
### Where do you get Azurite? (npm, DockerHub, NuGet, Visual Studio Code Extension)
Visual Studio Code Extension
### What's the Node.js version?
v8.10.0
### What problem was encountered?
AWS C++ SDK throws "map:at" exception as GetBlob response doesn't contain "x-ms-creation-time" header, 
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp#L6871
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp#L7243

which it has to contain according to the latest specification:
https://docs.microsoft.com/en-us/rest/api/storageservices/get-blob :
Response Headers: The response for this operation **includes** the following headers:
x-ms-creation-time | Version 2017-11-09 and newer. The date/time at which the blob was created. The date format follows RFC 1123.

### Steps to reproduce the issue?
```
2021-07-26T13:06:10.760Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 info DispatchMiddleware: Operation=Blob_Download
2021-07-26T13:06:10.760Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 verbose AuthenticationMiddlewareFactory:createAuthenticationMiddleware() Validating authentications.
2021-07-26T13:06:10.760Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 info PublicAccessAuthenticator:validate() Start validation against public access.
2021-07-26T13:06:10.760Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 debug PublicAccessAuthenticator:validate() Getting account properties...
2021-07-26T13:06:10.760Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 debug PublicAccessAuthenticator:validate() Retrieved account name from context: devstoreaccount1, container: kir1, blob: test/db_key
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 debug PublicAccessAuthenticator:validate() Skip public access authentication. Cannot get public access type for container kir1
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 info BlobSharedKeyAuthenticator:validate() Start validation against account shared key authentication.
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 info BlobSharedKeyAuthenticator:validate() [STRING TO SIGN]:"GET\n\n\n\n\n\n\n\n\n\n\n\nx-ms-client-request-id:19dac275-c6f6-4961-5845-790b437aae42\nx-ms-date:Mon, 26 Jul 2021 13:06:10 GMT\nx-ms-range:bytes=0-64\nx-ms-version:2020-02-10\n/devstoreaccount1/devstoreaccount1/kir1/test/db_key"
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 info BlobSharedKeyAuthenticator:validate() Calculated authentication header based on key1: SharedKey devstoreaccount1:kMqeNWWlocIjkJTblsJA2wVIJPhBGmT6x/VLoIlxsOc=
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 info BlobSharedKeyAuthenticator:validate() Signature 1 matched.
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 verbose DeserializerMiddleware: Start deserializing...
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 info HandlerMiddleware: DeserializedParameters={"options":{"range":"bytes=0-64","requestId":"19dac275-c6f6-4961-5845-790b437aae42","leaseAccessConditions":{},"cpkInfo":{},"modifiedAccessConditions":{}},"version":"2020-02-10"}
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 info BlobHandler:downloadBlockBlobOrAppendBlob() NormalizedDownloadRange=bytes=0-63 RequiredContentLength=64
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 debug OperationQueue.operate() Schedule incoming job a9083e49-7255-4e79-873d-7119f1e99d51
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 debug OperationQueue:execute() Current runningConcurrency:0 maxConcurrency:100 operations.length:1
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 verbose FSExtentStore:readExtent() Creating read stream. LocationId:Default extentId:3d5a4391-df66-45a7-ac05-0980726c8dc2 path:/home/hadoop/WORK/x100_test/srcs/trunk/__blobstorage__/3d5a4391-df66-45a7-ac05-0980726c8dc2 offset:0 count:64 end:63
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 debug OperationQueue.operate() Job a9083e49-7255-4e79-873d-7119f1e99d51 completes callback, resolve.
2021-07-26T13:06:10.761Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 verbose SerializerMiddleware: Start serializing...
2021-07-26T13:06:10.762Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 info Serializer: Start returning stream body.
2021-07-26T13:06:10.762Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 debug OperationQueue:execute() Current runningConcurrency:0 maxConcurrency:100 operations.length:0
2021-07-26T13:06:10.762Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 debug OperationQueue:execute() return. Operation.length === 0
2021-07-26T13:06:10.762Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 info EndMiddleware: End response. TotalTimeInMS=2 StatusCode=206 StatusMessage=Partial Content Headers={"server":"Azurite-Blob/3.13.1","last-modified":"Mon, 26 Jul 2021 10:50:08 GMT","content-length":"64","content-type":"application/octet-stream","content-range":"bytes 0-63/64","etag":"\"0x1A92FE5C900A6C0\"","content-md5":"KsVYZb7OqnS1xqLaUrzHAw==","x-ms-blob-type":"BlockBlob","x-ms-lease-state":"available","x-ms-lease-status":"unlocked","x-ms-client-request-id":"19dac275-c6f6-4961-5845-790b437aae42","x-ms-request-id":"e3e7999f-ab15-47d4-801c-2dfb26bcaa06","x-ms-version":"2020-08-04","accept-ranges":"bytes","date":"Mon, 26 Jul 2021 13:06:10 GMT","x-ms-server-encrypted":"true","x-ms-blob-content-md5":"KsVYZb7OqnS1xqLaUrzHAw=="}
2021-07-26T13:06:10.763Z e3e7999f-ab15-47d4-801c-2dfb26bcaa06 verbose FSExtentStore:readExtent() Read stream closed. LocationId:Default extentId:3d5a4391-df66-45a7-ac05-0980726c8dc2 path:/home/hadoop/WORK/x100_test/srcs/trunk/__blobstorage__/3d5a4391-df66-45a7-ac05-0980726c8dc2 offset:0 count:64 end:63
```

### Have you found a mitigation/solution?
It looks like "x-ms-creation-time" has to be described in mappers.ts;
https://github.com/Azure/Azurite/blob/main/src/blob/generated/artifacts/mappers.ts#L3359

**Comment by blueww** [comment]
@kshcherbatov

Thanks for raising the issue!
We will evaluate it.

**Comment by blueww** [comment]
The fix already released in Azurite 3.14.2 .
